### DISCUSSION TITLE
Reference system in Stellarium

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#2697] [Q&A]
I have been learning about ICRS, FK5 and other systems, and I wonder what is in use in Stelarium? Checking some stars with Simbad, seems that the coordinates J2000 in Stellarium do not match under any of those systems.

**Comment by gzotti** [comment]
Studying the source code, IIRC the reference frame is that of VSOP87. The question is always by how much there is a mismatch and how much this matters for your application.

**Reply by diegorodriguezperegrin** [reply]
I thought VSOP87 was a planetary theory. That is, a set of formulae to compute objects positions in the solar system only. 

There's an option to show a ICRS grid in the tool bar, and User manual 18.2 cites "Current catalogs and atlases use coordinates for the standard epoch J2000.0. The currently best deﬁned coordinate system, the International Celestial Reference Frame (ICRF), is one particular version described in detail in the astronomical literature", but is clear that Stellarium is not using it for some reason.

**Reply by gzotti** [reply]
Sure, but it uses one particular reference system that IIRC we also use. More: please read the code. The question is always by how much there is a mismatch and how much this matters for your application. Can you give a few deviation values as examples?

**Reply by diegorodriguezperegrin** [reply]
Adhara
06h58m37.35s       -28d58m03.4s  Stellarium 
06h58m37.54876s -28d58m19.5102 ICRS

21CVn
13h18m12.53s                +49d41m04.2s Stellarium
13h18m14.5095312384  +49d40m55.429204656s ICRS

Atria
16h48m38.50s                -69d01m53.7s Stellarium
16h48m39.89508            -69d01m39.7626 ICRS

**Reply by gzotti** [reply]
Just as VSOP is not a reference system in itself, ICRS is not a star catalog either. Our sources are documented in the User Guide.

**Reply by diegorodriguezperegrin** [reply]
Okay, I see that Stellarium uses Hipparcos (first three files), Tycho 2 (two intermediate files) and NOMAD (four last files) for  fainter stars. The newest of these three catalog may be 20 years old. Then, individual positions for each star are referred in each record to the central point in the star's zone, in both axis 1 and 2. "When parsing star catalogues, it is necessary to know about the zone model to be able to extract positional data", can be read in the User's Manual. Stellarium uses many (up tp 20·4^level) zones. So where can I read about how those zone models adhere to any particular reference frame?

**Reply by gzotti** [reply]
I see a few arcseconds mismatch, but I do not see how you have retrieved those data: program version, date set, with nutation, aberration?  

The star catalog has been designed before I joined the project in 2010, and the original contributor has left the project soon after without writing such valuable documentation. So it is not documented. Stellarium at that time was not (and therefore still is not) intended to be used as a high-precision astrometry tool, to calibrate professional observatory instrumentation or even a reference database for others to use. Appendix F (and a reference therein about accuracies of star catalogs in software) is all we have to offer.  Whoever wants to upgrade or just analyze the star catalog will have to do some research of his own. For the time being it is regarded good enough for identification.  Good luck.

**Comment by diegorodriguezperegrin** [comment]
I just sourced them from Simbad, ICRS coordinates. Stellarium resorts to
Simbad in some stances, too. Maybe for a future upgrade this topic may be
covered to provide a coherent documented reference system that probably
would not only eliminate discrepancies, but also reduce the workload of
future developers attempting to upgrade or extend the current star catalogs.

None of that can diminish a bit the enormous amount of knowledge already
poured into Stellarium at its present form. Many thanks to have had the
courage to take this project from the original contributors and take it to
the modern version, incredibly rich in features and constant improvement,
Georg!
